tdrs-backend/tdpservice/urls.py

```diff
@@ -38,10 +39,15 @@
 ]
 
 # Add 'prefix' to all urlpatterns to make it easier to version/group endpoints
+BASE_DIR = os.path.dirname(os.path.abspath(__file__))
 urlpatterns = [
     path("v1/", include(urlpatterns)),
     path("admin/", admin.site.urls, name="admin"),
-] + static(settings.STATIC_URL, document_root=settings.STATIC_ROOT)
+    path("", IndexView.as_view(), name="index"),
+]
```

Comment:
Add back the original static root, this was removed by mistake

Reply:
@riatzukiza please feel free to ping me when this has been addressed 😃

Reply:
I fixed these locally immediately after and I just forgot to push them 😅